com.cubrid.cubridmigration.console/pom.xml

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd"
+         xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>com.cubrid.cubridmigration</groupId>
+        <artifactId>cubridmigration</artifactId>
+        <version>1.0.0-SNAPSHOT</version>
+        <relativePath>../</relativePath>
+    </parent>
+    <version>1.0.0-SNAPSHOT</version>
+    <artifactId>com.cubrid.cubridmigration.console</artifactId>
+    <packaging>pom</packaging>
+
+    <properties>
+        <console-filename>cubridmigration-console</console-filename>
+    </properties>
+
+    <build>
+        <plugins>
+            <plugin>
+                <artifactId>maven-assembly-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>migrationconsole-tar</id>
+                        <phase>package</phase>
+                        <goals>
+                            <goal>single</goal>
+                        </goals>
+                        <configuration>
+                            <descriptors>
+                                <descriptor>${basedir}/console-tar.xml</descriptor>
+                            </descriptors>
+                            <finalName>${console-filename}</finalName>
+                            <archive>
+                                <manifest>
+                                    <addClasspath>true</addClasspath>
+                                </manifest>
+                            </archive>
+                        </configuration>
+                    </execution>
+                    <execution>
+                        <id>migrationconsole-zip</id>
+                        <phase>package</phase>
+                        <goals>
+                            <goal>single</goal>
+                        </goals>
+                        <configuration>
+                            <descriptors>
+                                <descriptor>${basedir}/console-zip.xml</descriptor>
+                            </descriptors>
+                            <finalName>${console-filename}</finalName>
+                            <archive>
+                                <manifest>
+                                    <addClasspath>true</addClasspath>
+                                </manifest>
+                            </archive>
+                        </configuration>
+                    </execution>
+                </executions>
+
+                <configuration>
+                    <tarLongFileMode>posix</tarLongFileMode>
```

Comment:
Strangely, I get an error when using 'posix'. 'In case of 'gnu', no error occurs.
Is 'posix' option working properly?

[Error Msg] : Failed to execute goal org.apache.maven.plugins:maven-assembly-plugin:2.2-beta-5:single (migration-linux) on project com.cubrid.cubridmigration.product: Failed to create assembly: Error creating assembly archive linux: posix is not a legal value for this attribute -> [Help 1]

Reply:
As you said, it works fine in version 3.9.6.

Since an error occurs in versions 3.8.1 and 3.6.3, please include MAVEN version information in the README.